Source/SwiftLintBuiltInRules/Rules/Style/SortedImportsRuleExamples.swift

```diff
@@ -0,0 +1,157 @@
+internal struct SortedImportsRuleExamples {
+    private static let groupByAttributesConfiguration = ["grouping": "attributes"]
+
+    static let nonTriggeringExamples = [
+        Example("import AAA\nimport BBB\nimport CCC\nimport DDD"),
+        Example("import Alamofire\nimport API"),
+        Example("import labc\nimport Ldef"),
+        Example("import BBB\n// comment\nimport AAA\nimport CCC"),
+        Example("@testable import AAA\nimport   CCC"),
+        Example("import AAA\n@testable import   CCC"),
```

Comment:
Would be easier if these examples (and the others below) would use text blocks too.

Reply:
Thank you for reformatting this!